tests/rptest/tests/datalake/datalake_e2e_test.py

```diff
@@ -134,6 +134,31 @@ def test_avro_schema(self, cloud_storage_type, query_engine):
                 assert spark_describe_out == spark_expected_out, str(
                     spark_describe_out)
 
+    @cluster(num_nodes=4)
+    @matrix(cloud_storage_type=[CloudStorageType.S3])
```

Comment:
nit: will the test fail or be skipped when running on e.g. azure?

Reply:
Ah yea updated to use the supported_storage_types()